var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -145,6 +145,40 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
         msg="Use paraview@5.9.0 with %xl_r. Earlier versions are not able to build with xl.",
     )
 
+    # CUDA ARCH
+
+    # This is (more or less) the mapping hard-coded in VTK-m logic
+    # see https://gitlab.kitware.com/vtk/vtk-m/-/blob/v2.1.0/CMake/VTKmDeviceAdapters.cmake?ref_type=tags#L221-247
+    supported_cuda_archs = {
+        "20": "fermi",
+        "21": "fermi",
+        "30": "kepler",
+        "32": "kepler",
+        "35": "kepler",
+        "37": "kepler",
+        "50": "maxwel",
+        "52": "maxwel",
+        "53": "maxwel",
+        "60": "pascal",
+        "61": "pascal",
+        "62": "pascal",
+        "70": "volta",
+        "72": "volta",
+        "75": "turing",
+        "80": "ampere",
+        "86": "ampere",
+    }
+
+    # VTK-m and transitively ParaView does not support Tesla Arch
+    for _arch in range(10, 14):
+        conflicts("cuda_arch=%d" % _arch, when="+cuda", msg="ParaView requires cuda_arch >= 20")
```

Comment:
```suggestion
        conflicts(f"cuda_arch={_arch}", when="+cuda", msg="ParaView requires cuda_arch >= 20")
```

Reply:
I generally prefer to not touch existing code (here I just moved it), but I agree that we should use f-string. Done! 😉